python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vxlan_interface.schema.yml

```diff
@@ -100,6 +102,16 @@ keys:
                       items:
                         type: str
                         description: Remote VTEP IP Address.
+              vlan_range:
+                type: dict
+                description: Set VNI on range of vlans.
+                keys:
+                  vlans:
+                    type: str
+                    description: e.g. 100-110
```

Comment:
I think we should give an example of comma(,) separated ranges in description as well.

Reply:
Done.